salt/matchers/glob_match.py

```diff
@@ -1,22 +1,22 @@
-# -*- coding: utf-8 -*-
 """
 This is the default glob matcher function.
 """
-from __future__ import absolute_import, print_function, unicode_literals
 
 import fnmatch
+import logging
 
-from salt.ext import six  # pylint: disable=3rd-party-module-not-gated
+log = logging.getLogger(__name__)
 
 
-def match(tgt, opts=None):
+def match(tgt, opts=None, minion_id=None):
     """
     Returns true if the passed glob matches the id
     """
     if not opts:
         opts = __opts__
-    minion_id = opts.get("minion_id", opts["id"])
```

Comment:
What is expecting `minion_id` to be in opts? If we remove this are we removing functionality from somewhere else?

Reply:
I believe this was something that was previously in place to support metaproxy proxies but I found a few references related to pillar so I updated the code.